managing_providers/_topics/automation_management_providers.md

```diff
@@ -539,3 +539,31 @@ Use the following steps to run the Terraform Template.
 
 The service item details can be viewed when you navigate to **Services** > **My Services in {{ site.data.product.title_short }}**.
 
+#### Retire the Terraform services
+
+When Terraform provisioned service resources are no longer required, the resources can be retired. The retirement action initiates the Terraform backend to run the `destroy` command. This action destroys all the remote resources that are managed by the Terraform configuration.
+
+Use the following steps to retire the service instance immediately:
+
+1. In the navigation bar, click **Service catalog** > **Services**.
```

Comment:
Saw this just now, it is not 

> ~~In the navigation bar, click **Service catalog** > **Services**~~

It should be 

> In the navigation bar, click **Services**

Reply:
You have to go to `Service Catalog` in order to access `Services`. Please confirm.